webhooks/mutatepod_parts.go

```diff
@@ -42,17 +42,23 @@ func k8sSATokenVolume(
 						Path:              K8sSATokenName,
 					},
 				}},
-				DefaultMode: pointer.Int32(288), // 440
+				DefaultMode: pointer.Int32(444),
```

Comment:
Why does this need this?

Reply:
Because 288 isn't translated to 440, it is literally placed as 288 and 8 isn't a valid permission value. we can set 440 if you prefer :)

Reply:
oh, sorry... Let's change it to `440`🙇

Reply:
Sure, I'll change it later on today

Reply:
I checked that `288` is equivalent with octal int `0440` in golang playground: https://go.dev/play/p/_ul6P7zpY1_6

So, we should 

```golang 
pointer.Int32(0440)
```

for readability.

Reply:
done